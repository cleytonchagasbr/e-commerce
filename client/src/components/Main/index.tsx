import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    <S.Logo src="/img/logo.svg" alt="Logo" />
    <S.Title>React Avan√ßado</S.Title>
    <S.Description>Typescript and NextJS</S.Description>
    <S.Ilustration src="/img/hero-illustration.svg" alt="Personagem" />
  </S.Wrapper>
)

export default Main
