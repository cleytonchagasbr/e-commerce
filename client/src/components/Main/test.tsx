import { screen, render } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should Render Header', () => {
    render(<Main />)

    expect(screen.getByRole('heading', { name: /react avan√ßado/i }))
      .toBeInTheDocument
  })
})
