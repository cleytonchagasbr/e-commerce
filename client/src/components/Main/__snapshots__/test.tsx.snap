// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="styles__Wrapper-hezPq bRacfh"
>
  <img
    alt="Logo"
    class="styles__Logo-bSrzMK cTOTrm"
    src="/img/logo.svg"
  />
  <h1
    class="styles__Title-cLvJzd eXdJVt"
  >
    React Avan√ßado
  </h1>
  <h2
    class="styles__Description-elWNsj fjFURb"
  >
    Typescript and NextJS
  </h2>
  <img
    alt="Personagem"
    class="styles__Ilustration-bZuImV hbiFlf"
    src="/img/hero-illustration.svg"
  />
</main>
`
